antarest/study/common/studystorage.py

```diff
@@ -248,7 +248,7 @@ def export_output(self, metadata: T, output_id: str, target: Path) -> None:
     def export_study_flat(
         self,
         metadata: T,
-        dest: Path,
+        dst_path: Path,
```

Comment:
I would have used a more descriptive name here like `target_path`, "dst" can be confusing 
note: same for other abbreviations elsewhere